src/StockServiceManager.php

```diff
@@ -31,13 +32,30 @@ class StockServiceManager implements StockServiceManagerInterface, StockTransact
   protected $configFactory;
 
   /**
+   * The current store.
+   *
+   * @var \Drupal\commerce_store\Entity\Store
+   */
+  protected $currentStore;
+
+
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $currentUser;
+
+  /**
    * Constructs a StockServiceManager object.
    *
    * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
    *   The config factory.
    */
   public function __construct(ConfigFactoryInterface $config_factory) {
     $this->configFactory = $config_factory;
+    $this->currentStore = \Drupal::service('commerce_store.current_store')->getStore();
```

Comment:
The Stock Level Formatter does not work because **\Drupal::service('commerce_store.current_store')->getStore()** dose not return the correct current store